men_team_1/app/controllers/users_controller.rb

```diff
@@ -21,6 +21,15 @@ def create
 
   def show
     @user = User.find(params[:id])
+
+    @request = []
```

Comment:
`@request` is not a great name, especially in a web app. Usually it would refer to the web request. Be more specific in your naming to improve readability.

In this case, I might call it e.g. `@pending_mentees`.